src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -274,9 +274,7 @@ rec_free(struct d_hash_table *htable, d_list_t *rlink)
 
 	rc = dfs_release(hdl->oh);
 	if (rc == ENOMEM)
-		rc = dfs_release(hdl->oh);
-	if (rc)
-		D_ERROR("dfs_release() failed: %d (%s)\n", rc, strerror(rc));
+		dfs_release(hdl->oh);
```

Comment:
Was there a problem with the code here before?

It's always a good idea to review logging and see if it's appropriate so if you're removing logging as you think it improves the end result that's fine.  The presence of logging itself however will not cause failures in NLT, as long as the messages are well-formed which this one looked to be.

Reply:
The log here does not cause any failure. I will put it back. Thank you!